src/runtime/account.ts

```diff
@@ -45,23 +70,97 @@ export class Account {
   }
 
   get accountId(): string {
-    return this.najAccount.accountId;
+    return this._accountId;
+  }
+
+  get prefix(): string {
+    return this.levelUp ? this.accountId.replace(`.${this.levelUp}`, "") : this.accountId;
```

Comment:
Why is this `levelUp` logic here? I don't see it used in this PR.

Reply:
It was the only way I could solve the issue of passing arguments with the passing of accounts.  And we needed this because we needed to pass runtime to accounts.  I know it's a tad confusing.